src/components/ImageGalleryModal/_image-tile.scss

```
@@ -0,0 +1,95 @@
+@import '../../globals/vars';
+
+$title-height: 3rem;
+
+@mixin image-container($max-height, $max-width) {
+  height: $max-height;
+  width: $max-width;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  img {
+    max-width: $max-width;
+    max-height: $max-height;
+  }
+}
+
+.#{$iot-prefix}--image-tile {
+  $image-max-height: 12rem;
+  $image-max-width: 20rem;
+
+  background-color: $ui-01;
+  width: $image-max-width;
+  height: calc(#{$image-max-height} + #{$title-height});
+  padding: 0;
+
+  .#{$iot-prefix}--image-tile__image-container {
+    @include image-container($image-max-height, $image-max-width);
+  }
+}
+
+.#{$iot-prefix}--image-tile--wide {
+  $image-max-height: 3rem;
+  $image-max-width: 5rem;
+
+  width: 100%;
+  height: $title-height;
+  padding: $spacing-03;
+  border-top: 1px solid $ui-03;
+
+  &:nth-of-type(1) {
+    border-color: transparent;
+  }
+  // The border must be hidded when focused or selected, also for sibling
+  &.#{$prefix}--tile--is-selected,
+  &:focus {
+    border-color: transparent;
+    & + input + .bx--tile {
```

```suggestion
    & + input + .#{$prefix}--tile {
```

just a missing prefix

---

Will fix.